api/s2n.h

```diff
@@ -438,8 +438,8 @@ extern int s2n_get_cert_chain_length(struct s2n_cert_chain_and_key *chain_and_ke
 /**
  * Returns the certificate `out_cert` present at the index `cert_idx` of the certificate chain `chain_and_key`.
  * 
- * Note that the index of the `head_cert` is zero. If the certificate chain `chain_and_key` is NULL or the
- * certificate index value is not in the acceptable range for the input certificate chain, an error is thrown.
+ * Note that the index of the root certificate is zero. If the certificate chain `chain_and_key` is NULL or the
```

Comment:
Are you sure? In most things I've seen the leaf certificate is first and the root certificate is last.

Reply:
Nice catch `head_cert` refers to the first leaf cert within s2n_cert_chain_and_key structure and not the root cert. Fixing it.